tools/ci/run_tc.py

```diff
@@ -216,6 +216,8 @@ def start_xvfb():
 def get_extra_jobs(event):
     body = None
     jobs = set()
+    if not event:
+        return jobs
```

Comment:
IIUC, you're trying to make `run_tc.py` work locally, without the various env vars?

I'm not sure if that's a good idea. Isn't `tools/docker` supposed to be the entrypoint for that use case?

Reply:
Well I found it pretty useful for it to run locally when testing this and various other things. It's pretty convenient to be able to run a single command and have all the chrome install and Xvfb stuff taken care of.